crates/welcome/src/welcome.rs

```diff
@@ -160,24 +167,56 @@ impl Render for WelcomePage {
                                                     .ok();
                                             })),
                                     )
-                                    .child(
-                                        Button::new(
-                                            "sign-in-to-copilot",
-                                            "Sign in to GitHub Copilot",
-                                        )
-                                        .icon(IconName::Copilot)
-                                        .icon_size(IconSize::XSmall)
-                                        .icon_color(Color::Muted)
-                                        .icon_position(IconPosition::Start)
-                                        .on_click(
-                                            cx.listener(|this, _, cx| {
-                                                this.telemetry.report_app_event(
-                                                    "welcome page: sign in to copilot".to_string(),
-                                                );
-                                                copilot::initiate_sign_in(cx);
-                                            }),
-                                        ),
-                                    )
+                                    .child({
+                                        if cx.has_flag::<ZedPro>() && self.is_signed_out {
```

Comment:
Does this make sense? I followed what we do in the Assistant Panel, but how can a feature flag be set while signed out?

Reply:
> Can a feature flag really be set while signed out?

No, I don't think so. I think in this case, we might want to wait with this change for the public beta and not the private beta?

Reply:
Cool! I'll remove the FF check and move this PR to draft